visualizer/data-tree.js

```diff
@@ -30,6 +30,9 @@ class DataTree {
     this.maxRootAboveMean = 0
     this.maxRootBelowMean = 0
 
+    // Set by countTotalFrames
```

Comment:
Hmm... I can't see where `countTotalFrames` is defined

Reply:
Good spot, a hangover from a different approach 👍